CMakeLists.txt

```
@@ -46,9 +46,15 @@ else (WIN32)  # assuming Linux
 
 endif (WIN32)
 
-add_library (${PROJECT_NAME} ${SOURCES})
+add_library(${PROJECT_NAME} STATIC ${SOURCES})
```

```suggestion
add_library(${PROJECT_NAME} ${SOURCES})
```
It's more flexible if we don't set this here.